### ISSUE TITLE
Inconsistent use of apply button in local services window

### ISSUE THREAD

**Issue by MartinGebert** [blueman-project/blueman#2162]
blueman:
BlueZ:
Distribution:
Desktop environment:

* [X] I have consulted the [Troubleshooting page](https://github.com/blueman-project/blueman/wiki/Troubleshooting) and done my best effort to follow.

The *Local Services* window features an Apply button on both pages ("Network", "Transfer") available on my system, which is enabled after opening the window. It gets properly enabled after changing any configuration on the "Transfer" page, serving it's purpose to save the changes.

In the "Network" page only the settings "PAN Support" and "DUN Support" are enabled/available for me, the widgets in "Services" are disabled. Changing any of the PAN or DUN settings doesn't enable the Apply button. However closing the window and opening it again shows the changes apparently to be applied anyhow. Compare this screenshot after switching both radio buttons; I would expect the Apply button to be enabled at this point:

![image](https://github.com/blueman-project/blueman/assets/31990833/c2315ae1-be67-4760-9786-bc36213ee108)

**Comment by infirit** [comment]
Yes it is inconsistent. Changing those is instant apply but they can behave like the rest without much issues (I hope :rofl: ). Marked as enhancement.
